systemservices/marketplace/mesg.yml

```diff
@@ -11,129 +11,107 @@ configuration:
               owner: &owner
-                name: ""
-                description: ""
                 type: String
               versions: &versions
-                name: ""
-                description: ""
                 type: Object
                 repeated: true
                 optional: true
                 object: &version
                   versionHash: &versionHash
-                    name: ""
-                    description: ""
                     type: String
                   manifest: &manifest
-                    name: ""
-                    description: ""
                     type: String
                   manifestProtocol: &manifestProtocol
-                    name: ""
-                    description: ""
                     type: String
                   manifestData: &manifestData
-                    name: ""
-                    description: ""
                     type: Object
                     optional: true
-                    object: 	
-                      version: 	
-                        name: ""	
-                        description: ""	
-                        type: String	
+                    object:
+                      version:
+                        type: String
                       service: &manifestServiceData	
-                        type: Object	
-                        object:	
-                          hash:	
-                            name: ""	
-                            description: ""	
-                            type: String	
-                          hashVersion: 	
-                            name: ""	
-                            description: ""	
-                            type: String	
-                          deployment:	
-                            type: Object	
-                            object:	
-                              type: 	
-                                name: ""	
-                                description: ""	
-                                type: String	
-                              source: 	
-                                name: ""	
-                                description: ""	
-                                type: String	
-                          definition: 	
-                            name: ""	
-                            description: ""	
-                            type: Object	
-                          readme: 	
-                            type: String	
+                        type: Object
+                        object:
+                          hash:
+                            type: String
+                          hashVersion:
+                            type: String
+                          deployment:
+                            type: Object
+                            object:
+                              type:
+                                type: String
+                              source:
+                                type: String
+                          definition:
```

Comment:
I don't think we should add a "partial" validation for _definition_. 
to me, it's an all-or-nothing thing.

Reply:
Done